setup.py

```diff
@@ -164,9 +164,15 @@ def main():
             "-DBUILD_PNG=ON",
         ]
         + (
+            # CMake flags for windows/arm64 build
+            ["-DCMAKE_GENERATOR_PLATFORM=ARM64",
```

Comment:
This like breaks regular x86 configuration.

Reply:
Thanks for the review! Can you explain how it would break x86? It is not clear to me.

Reply:
Sorry, my fault.